test/simple_real_world_projects/data/sample_autotools_project/Makefile.in

```diff
@@ -0,0 +1,769 @@
+# Makefile.in generated by automake 1.16.3 from Makefile.am.
+# @configure_input@
+
+# Copyright (C) 1994-2020 Free Software Foundation, Inc.
+
+# This Makefile.in is free software; the Free Software Foundation
+# gives unlimited permission to copy and/or distribute it,
+# with or without modifications, as long as this notice is preserved.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY, to the extent permitted by law; without
+# even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.
```

Comment:
*cries in cmake*

Is the only way in which we can do this to commit this file?
I'm really not very happy to commit this stuff. Buf if so, be it.

Reply:
After private discussion we decided to keep the generated makefile to not increase the testsuite runtime and not adding autoconf as a requirement.